src/components/UI/Form/StyledInput.js

```diff
@@ -1,22 +1,32 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 const Input = styled.input`
-	text-transform: uppercase;
-	background-color: var(--main-bg-color-dark);
-	color: var(--text-color-light);
-	border: none;
-	height: 38px;
-	border-radius: 19px;
-	padding: 10px;
+	${({ variant = 'text' }) =>
+		variant === 'text' &&
+		css`
+			text-transform: uppercase;
+			background-color: var(--main-bg-color-dark);
+			color: var(--text-color-light);
+			border: none;
+			height: 38px;
+			border-radius: 19px;
+			padding: 10px;
 
-	:focus {
-		outline: solid 3px var(--accent-color);
-	}
+			:focus {
+				outline: solid 3px var(--accent-color);
+			}
 
-	::placeholder {
-		color: var(--text-color-light);
-		opacity: 1;
-	}
+			::placeholder {
+				color: var(--text-color-light);
+				opacity: 1;
+			}
+		`}
+
+	${({ variant }) =>
+		variant === 'file' &&
+		css`
+			display: none;
```

Comment:
Das würde ich vielleicht anders lösen, denn dein Input wird jetzt von Voice Overs übersprungen, weil display:none den aus dem Document-Flow rausnimmt

Reply:
changed it to `position: fixed;
			top: -100%;
			left: -100vw;`